ivi-layermanagement-api/ilmControl/src/writepng.h

```diff
@@ -0,0 +1,32 @@
+/***************************************************************************
+ *
+ * Copyright (C) 2020 Advanced Driver Information Technology Joint Venture GmbH
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+#ifndef IVICONTROLLER_WRITEPNG_H_
+#define IVICONTROLLER_WRITEPNG_H_
+
+#include <stdint.h>
+
+int
+save_as_png(const char *filename,
+            const char *buffer,
+            int32_t width,
+            int32_t height,
+            int16_t bpp,
+            int has_alpha);
+
+#endif /* IVICONTROLLER_WRITEPNG_H_ */
```

Comment:
@rbmeftah 
Please add a newline at the end of `writepng.h` like in `writepng.c`.